glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+  + #### *DTLS*([Datagram Transport Layer Security](https://developer.mozilla.org/en-US/docs/Glossary/DTLS)) is a protocol used for providing security to datagram-based applications. It is based on TLS and guarantees a similar level of security. All of the WebRTC related protocols are required to encrypt their communications using DTLS, this includes [SCTP](https://developer.mozilla.org/en-US/docs/Glossary/SCTP), [SRTP](/glossary/glossary#RTP) and [STUN](/glossary/glossary#STUN).
++ #### *Container format* is a file format that allows multiple data streams to be embedded into a single file, e.g. MP4 format can contain video, audio, and subtitles streams inside of it.
++ #### *YUV* is a color encoding system that defines one [luminance](https://en.wikipedia.org/wiki/Luminance) and two [chrominance](https://en.wikipedia.org/wiki/Chrominance) components. By reducing the resolution of the chrominance components it is possible to compress an image with minuscule effect on human perception of the image. 
++ #### *Encoding* is a process of converting media from raw format to encoded format. The main purpose is to reduce media size - the raw format is uncompressed and takes up a lot of space. Examples of encoded formats are [MP3](https://en.wikipedia.org/wiki/MP3) and [AAC](https://en.wikipedia.org/wiki/Advanced_Audio_Coding) for audio and [AVC](https://en.wikipedia.org/wiki/Advanced_Video_Coding) and [MPEG4](https://en.wikipedia.org/wiki/MPEG-4_Part_2) for video.
++ #### *Decoding* is a process of converting media from encoded format to raw format, e.g. in order to play it on the end device.
++ #### *Encryption* is a way of modifying a message, so that only authorized parties are able to interpret it.
++ #### *Decryption* is a process of retrieving data from an encrypted message.
++ #### *Muxing*(abbr. from multiplexing) is a method of combining multiple streams into a single container, e.g. muxing video and audio into an MP4 container.
++ #### *Demuxing*(abbr. from demultiplexing) is a method of separating streams from one combined container, e.g. retrieving audio and video from MP4.
++ #### Server's architecture: ([here](https://millo-l.github.io/WebRTC-implementation-method-Mesh-SFU-MCU/) is a short article to get you started)
+  + #### *SFU*([Selective Forwarding Unit](https://millo-l.github.io/WebRTC-implementation-method-Mesh-SFU-MCU/#22-sfuselective-forwarding-unit-server)) is a video conferencing architecture that consists of a single server, which receives incoming streams from all participants and forwards each participant's stream to all other conference participants.
+  + #### *MCU*([Multipoint Control Unit](https://millo-l.github.io/WebRTC-implementation-method-Mesh-SFU-MCU/#23-mcumulti-point-control-unit-server)) is an architecture consisting of a single server, which receives incoming streams from all participants, multiplexes the streams, and sends them to each of the participants.
```

Comment:
I think it is quite important to note that we are mixing the stream (therfore we are getting, i.e. a single HD stream out of 10 HD streams from each of the peers) instead of multiplexing them (which, as we have agreed, is a fully inversible process)